docs/architecture/decisions/adr-004.md

```diff
@@ -0,0 +1,33 @@
+[accessibility support baseline]: https://www.w3.org/TR/WCAG-EM/#step1c
+
+# ADR 4: Model browser specific values and computations
+
+## Context
+
+Having decided on [ADR 2](adr-002.md), we will no longer be tied to any one specific browser and are left with the choice of what to do about things that browsers are not aligned on. For example, if Alfa needs to implement an API to perform a given task and a given set of browsers do not agree on how the API should be implemented, Alfa can do several things:
+
+1. Implement the API the way it is implemented in the _least_ capable browser, in terms of the API, among a given set of browsers.
```

Comment:
Might just be me, but is it clear enough what is meant by "least capable" and "most capable"? 

Should this perhaps be defined somewhere? Or is it?